nautobot/circuits/filters.py

```diff
@@ -69,13 +69,15 @@ class Meta:
     def search(self, queryset, name, value):
         if not value.strip():
             return queryset
+        # pylint: disable=unsupported-binary-operation
```

Comment:
Can we add a TODO and reference to https://github.com/PyCQA/pylint/issues/7953 to each of these disables as a breadcrumb?

Reply:
Or possibly PyCQA/pylint#7381.